modules/tests/test/testconfigs/pipeline-test-config.go

```diff
@@ -0,0 +1,46 @@
+package testconfigs
+
+import (
+	. "github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/constants"
+	"github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/framework/testoptions"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type PipelineRunData struct {
```

Comment:
Do we want to support Workspaces in the future?

Reply:
We can add support some time later as now there is no need for it